functions/src/modules/SE.functions.ts

```diff
@@ -88,3 +88,58 @@ export const processNewAllotment = functions.database
     });
     return 1;
   });
+
+const serialRegex = "^[a-zA-Z]+-\\d+";
+
+const updateChunk = async (path: string, taskId: string) => {
+  return await db.ref(path).update({
+    taskId
+  });
+};
+
+const makeTaskId = (fileName: string, index: number) => {
+  return `${fileName.match(serialRegex)[0]}-${index}`;
+};
+
+export const createTaskFromChunks = functions.database.ref(
+    "/sound-editing/chunks/{listId}/{fileName}/{index}"
+).onCreate(async (snapshot, { params: { fileName, listId, index } }) => {
+  const {
+    continuationTo,
+    continuationFrom,
+    processingResolution = "",
+    taskId,
+    beginning,
+    ending
+  } = snapshot.val();
+  if (continuationTo || processingResolution.toLowerCase() !== "ok" || taskId) return;
+  const chunkDuration = ending - beginning;
+  const chunkTaskId = makeTaskId(fileName, index);
```

Comment:
Logic here is not correct.
Index of the chunk within a file cannot be used directly as a counter for the taskId.
See for example `ML1-069-3` task in the picture below. This task originates from the first chunk of the `ML1-069B` file, but it has counter `3`.
![Naming scheme](https://trello-attachments.s3.amazonaws.com/5b84c9b8e98b56066fe1f20b/5c1a3b232754818606f7b9c4/ab614bdc9daeb79992b7dfaeb8d2690f/IMG_5188183843007123015.jpg)

Reply:
And it will be more clear to name it
```suggestion
  const taskId = makeTaskId(fileName, index);
```

Reply:
It is important to test that this task id does not exist in the database, otherwise we should issue an error message and exit.